doc/nrf/protocols/matter/getting_started/adding_clusters.rst

```diff
@@ -12,6 +12,8 @@ As part of this guide, you will modify the :ref:`Matter template <matter_templat
 The sensor will periodically generate the simulated temperature sensor value and update the corresponding cluster attributes.
 This application will form a Matter device within a Matter network.
 
+In the bottom of the page, you can find the :ref:`guide <ug_matter_creating_accessory_vendor_cluster>` on how to add a custom vendor-specific cluster to the Matter application.
```

Comment:
I don't know whether we have to mention it's vendor-specific, as custom already covers it's all "customity" .
```suggestion
In the bottom of the page, you can find the :ref:`guide <ug_matter_creating_accessory_vendor_cluster>` on how to add a custom cluster to the Matter application.
```